var/spack/repos/builtin/packages/bzip2/package.py

```diff
@@ -46,6 +46,14 @@ def patch(self):
             filter_file('-Wall -Winline', '-Minform=inform', 'Makefile')
             filter_file('-Wall -Winline', '-Minform=inform', 'Makefile-libbz2_so')  # noqa
 
+        # The Makefiles use GCC flags that are incompatible with XL and nvcc
+        if self.compiler.name == 'xl':
+            filter_file('-fpic', '', 'Makefile')
```

Comment:
Isn't the XL equivalent flag `-fPIC`?

Reply:
@ax3l The makefile compile rule contains both `-fpic -fPIC`, so this line filters out the first of those. Technically, at least currently, XL ignores the `-fpic` so line 51 could be removed...